apis/Google.Cloud.Logging.NLog/docs/index.md

```diff
@@ -0,0 +1,36 @@
+{{title}}
+
+`Google.Cloud.Logging.NLog` is a .NET client library to integrate  [Google Stackdriver
+Logging](https://cloud.google.com/logging/) with
+[NLog](http://nlog-project.org/).
```

Comment:
Unwrap onto the previous line? It's pretty short.

Reply:
done